assets/TenancyServiceProvider.stub.php

```diff
@@ -40,7 +40,13 @@ public function events()
             Events\TenantSaved::class => [],
             Events\UpdatingTenant::class => [],
             Events\TenantUpdated::class => [],
-            Events\DeletingTenant::class => [],
+            Events\DeletingTenant::class => [
+                JobPipeline::make([
+                    Jobs\DeleteDomains::class,
+                ])->send(function (Events\TenantDeleted $event) {
+                    return $event->tenant;
```

Comment:
Actually one more thing I noticed here — the event is `Events\DeletingTenant` and yet we're typehinting `Events\TenantDeleted`. Why is that?

Reply:
> Why is that?

It's a mistake. It should be `DeletingTenant`.